web/src/stream_popover.ts

```diff
@@ -500,6 +504,90 @@ export async function build_move_topic_to_stream_popover(
             is_disabled;
     }
 
+    // Warn if any of current topic participants are NOT subscribed
+    // to the destination stream.
+    function warn_unsubscribed_participants(destination_stream_id: number): void {
+        $("#move_topic_modal .unsubscribed-participants-warning").remove();
+
+        const conversation_messages = message_util.get_messages_in_topic(
```

Comment:
This code should be aware that `message_util.get_messages_in_topic` only fetches locally cached messages; probably you should call `conversation_messages` => `locally_cached_conversation_messages`

Reply:
Ok, But the namings in [get_messages_in_topic](https://github.com/zulip/zulip/blob/eaeed0e0f877dca7d3fe3dd0d7c0680a3a692cf3/web/src/message_util.ts#L36-L45) code, don't tell they are cached really

Reply:
Yeah, `get_messages_in_topic` function should be renamed; you can add a follow-up commit to say `message_util.get_loaded_messages_in_topic`; we've been using that term elsewhere. 

For this application, I think we're OK with only looking at the locally available information; if we're missing some very long ago participation, the notice may not be relevant anyway. But it should be commented for sure, and we should use clear variable names.